clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -119,6 +128,63 @@ class ClangToCIRArgMapping {
 
 } // namespace
 
+cir::FuncType CIRGenTypes::getFunctionType(const CIRGenFunctionInfo &fi) {
```

Comment:
This should be refactored to reflect the changes in https://github.com/llvm/clangir/pull/1604. I apologize for the extra work this causes you. I didn't realize the extent of the changes that I was going to make there.